WildFly_28.0.0.Final/microprofile-6.0-selected-specifications/microprofile-6.0-selected-specifications-certification.adoc

```diff
@@ -0,0 +1,31 @@
+= TCK Results
+:ext-relative: {outfilesuffix}
+:imagesdir: ../../images/
+
+image:splash_wildflylogo_small.png[WildFly, align="center"]
+
+[[introduction]]
+== Introduction
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for the selected specifications from the release of https://github.com/eclipse/microprofile/releases/tag/6.0[MicroProfile 6.0]. This certification contains all specifications defined in https://github.com/eclipse/microprofile/releases/tag/6.0[MicroProfile 6.0] except for MicroProfile Metrics which is explicitly excluded as part of the Red Hat position statement.
+
+=== WildFly Full Platform, Certification Summary
+----
+Date:  Fri Apr 21 01:19:49 PM CEST 2023
+Product Name:  WildFly
+Product Version: WildFly 28.0.0.Final
+URL to product download page: https://wildfly.org/downloads
+
+Product Description: An implementation of the Jakarta EE Full Platform, Web Profile, and MicroProfile Specifications
+Java SE provider and Version tested with:  OpenJDK 11.0.18 17.0.6 
+OS version: Linux 6.2.9-703.inttf.fc37.x86_64 GNU/Linux
+----
+Specification Name, Version and download URL:
+https://download.eclipse.org/microprofile/microprofile-6.0/microprofile-spec-6.0.pdf[MicroProfile 6.0]
```

Comment:
I'm not sure we should include this, as we don't claim compatibility. The link in the Introduction feels different as that is providing context to the reader, whereas this is in the more formal part of how an impl declares compatibility.

Reply:
Otherwise, we would need to list all individual specs except metrics with their versions. I think we can keep it, and I'll add (except MicroProfile Metrics) to the definition.